src/Common/src/Interop/Unix/libcrypto/Interop.EVP.Cipher.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class libcrypto
+    {
+        // This was computed by sizeof(EVP_CIPHER_CTX) on a 64-bit Ubuntu
+        // machine running OpenSSL 1.0.1f.  If we end up making a native
```

Comment:
"If we end up making"... we have that now, right?  Do we have an issue tracking moving whatever from here you want moved?